src/agents/utility.jl

```diff
@@ -2227,7 +2269,7 @@ function solve_agent_problem!(
     return compute_difference_percentage_maximum_one_norm([
         (x_R_before, utility.x_R_my),
         (x_C_before, utility.x_C_my),
-    ])
+    ]), viu_obj_value
```

Comment:
I think this is the first time I've seen the second result being passed. How is it used/who uses it?

Reply:
This is used in DER aggregator's problem when constructing the cost-saving function for different levels of DER aggregation.

Reply:
Maybe DER aggregator should access this data from the agent, rather than as a return value?

Reply:
Elaine to address this one

Reply:
@elainethale Let's table this item. I was running simulations and realized some of the results should _not_ be updated when running the VIU CEM in DER aggregator's problem when constructing cost saving curves (results like investment and retirement should only be updated when running VIU's problem in its own module). If we somehow save the objective value as part of the VIU agent's data, I'm a little concerned it will create confusion as to which runs the objective value is associated with. We can chat more if this doesn't make sense.

Reply:
I really don't like adding a whole other return value for this one corner case. Maybe adding a new variable would be okay if I give it a really obvious "only for DERA" name?

Reply:
Ok, if you prefer that way, maybe we should also adding terms like "temp" and have comments that stress this value is only relevant when the VIU CEM is immediately called right beforehand.

Reply:
I named it `_obj_value`, with the initial underscore meant to indicate "not for general use." Feel free to name it something else if you would prefer.